web/src/hooks/__tests__/useUserLocation.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, waitFor } from '@testing-library/react'
+import React from 'react'
+
+import useUserLocation from '../useUserLocation'
+
+jest.mock('../useUserLocation', () => ({
+  __esModule: true,
+  default: jest.fn(),
+  getUserLocation: jest.fn(),
+}))
+
+const TestComponent = () => {
+  const { data } = useUserLocation()
+  return <div>{data?.coordinates?.join(',') ?? 'null'}</div>
+}
+
+describe('useUserLocation', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('should show coordinates when available', async () => {
+    ;(useUserLocation as jest.Mock).mockReturnValue({
```

Comment:
:wrench: Please change this to the following, same for other occurrences:
```suggestion
    mocked(useUserLocation).mockReturnValue({
```